public/data/food/processes/builder.json

```diff
@@ -4,594 +4,489 @@
     "unit": "kilogram",
```

Comment:
Seems we still have this prettier formatting differently issue between the two of us :(